### ISSUE TITLE
Add a post property for featured image, please

### ISSUE THREAD

**Issue by guillermomf2001** [vaakash/git-it-write#12]
It would be cool if anybody could add a feature image directly from post properties into the Yaml configguration of the post .md file. Something like this: 

```
---

title: Title of this post
menu_order: 1
post_status: draft
post_excerpt: Excerpt of this post.
post_featured_image: /_images/featured_image.jpg  # <-- featured image URL in a new yaml post property
taxonomy:
    category:
        - Javascript
    post_tag:
        - Javascript-config
        - VSCode

---

# Title post

The rest of the content...
```

**Comment by vaakash** [comment]
Hi @ZonaGuillermo,

There were requests for the same and a merge request for this too.
I'll review and implement the best practice to set featured image for the post.

Thanks,
Aakash

**Comment by myannou** [comment]
Thank you for this PR
Will this PR integrated in the next release ?

**Comment by vaakash** [comment]
Hi @myannou, I'm reviewing the changes. I did not receive a PR. I have some comments on the changes. I'll discuss and work on this further. I'll try to publish a version by this month end.

**Comment by vaakash** [comment]
This is addressed in v1.7. Marking this as closed.

**Comment by myannou** [comment]
I confirm this work correctly
Thanks a lot
